pkg/apis/workflow/register.go

```diff
@@ -2,10 +2,15 @@ package workflow
 
 // Workflow constants
 const (
-	Kind      string = "Workflow"
-	Group     string = "argoproj.io"
-	Singular  string = "workflow"
-	Plural    string = "workflows"
-	ShortName string = "wf"
-	FullName  string = Plural + "." + Group
+	Group                     string = "argoproj.io"
+	WorkflowKind              string = "Workflow"
```

Comment:
Is this a breaking change?
AFAIK, Go package versioning rules strongly discourage making breaking changes. Especially in point releases.

https://golang.org/doc/faq#get_version

>the import compatibility rule: “If an old package and a new package have the same import path, the new package must be backwards compatible with the old package.”

>don't remove exported names, encourage tagged composite literals, and so on. If different functionality is required, add a new name instead of changing an old one. If a complete break is required, create a new package with a new import path.

Can you tell us a bit more about the emergency circumstances that required renaming these variables?